### PR TITLE
SITs becoming mentors - remove separate journey

### PR DESCRIPTION
**Pull request by lazydevorg**
### Context

SITs becoming mentors - remove separate journey

- Ticket: CST-1799

### Changes proposed in this pull request

- SITs becoming mentors - add speedbump page (CST-1796)
- SITs can't accidentally add themselves as ECTs page (CST-1807)
- Remove "Add yourself as a mentor" option from add participant wizard (CST-1808)
- SITs becoming mentors - dedupe when registered at a previous school with a different email address (CST-1797)

image

image

image

### CODE REVIEW

File: app/views/schools/add_participants/cannot_add_yourself_as_ect.html.erb
Comment (Lloyd-K) [inline app/views/schools/add_participants/cannot_add_yourself_as_ect.html.erb:11]:
```suggestion
    <p class="govuk-body">If you’re trying to add yourself as a mentor, follow the link below.</p>
```

File: app/views/schools/add_participants/start_term.html.erb
Comment (Lloyd-K) [inline app/views/schools/add_participants/start_term.html.erb:24]: If possible, we'd use the contraction 'you’ll' here as it's easier for users to read. Is that a problem given that we may display the user's name instead? Thanks

**Review by Lloyd-K — CHANGES_REQUESTED** [review]

**Review by Lloyd-K — APPROVED** [review]

**Review by frankieroberto — APPROVED** [review]

**Review by tonyheadford — APPROVED** [review]
As far as I can see it looks OK, needs an OK from product before merging though to ensure the journey is correct. Be good to get this out!

**Pull request merged**
